EIPS/eip-606.md

```diff
@@ -16,8 +16,8 @@ This specifies the changes included in the hard fork named Homestead.
 
 - Codename: Homestead
 - Activation:
-  - Block >= 1,150,000 on Mainnet
-  - Block >= 494,000 on Morden
+  - Block >= 1,150,000 on Mainnet   (CHAIN_ID = 1)
+  - Block >= 494,000 on Morden      (CHAIN_ID = 2)
```

Comment:
I don't think it makes sense stating `chain_id` for block numbers when `chain_id` was not present. Don't even think `morden` has one since it was around when `chain_id` was introduced.

Adding it to the other EIPs where it was present may make sense though.

Reply:
Yeah, fair enough, although it should have a chain_id now, as well as the mainnet, as listed here: https://github.com/ethereum/EIPs/blob/master/EIPS/eip-155.md#list-of-chain-ids.

Reply:
But as you see it doesn't exists anymore and a different chain took up the value :)